x/rollapp/keeper/msg_server_update_state.go

```diff
@@ -83,6 +81,15 @@ func (k msgServer) UpdateState(goCtx context.Context, msg *types.MsgUpdateState)
 		FinalizationQueue: newFinalizationQueue,
 	})
 
+	// check if sequencer rotation in progress
+	nextAddr, found := k.sequencerKeeper.GetNextProposerAddr(ctx, msg.RollappId)
+	res := &types.MsgUpdateStateResponse{
+		NextProposerAddr:   nextAddr,
+		RotationInProgress: found,
+	}
```

Comment:
would document here or add ADR faq section/linkt to why we do it like this instead of event

Reply:
removed